gbpservice/nfp/configurator/api/v1/controllers/controller.py

```diff
@@ -322,10 +327,13 @@ def pull_notifications(self):
             self.channel.close()
             return notifications
         except pika.exceptions.ConnectionClosed:
+            LOG.error("Caught ConnectionClosed exception."
+                      "Creating new connection")
             self.create_connection()
```

Comment:
Connection getting closed is a serious problem. Is this happening in our testing?